hack/generated/pkg/armclient/template_client.go

```diff
@@ -142,6 +143,20 @@ func AuthorizerFromEnvironment() (autorest.Authorizer, error) {
 		return nil, err
 	}
 
+	// the previous never returns an error, so we must do
```

Comment:
File an issue for them, or find an existing issue and link against it possibly?

Reply:
@matthchr the way the API works it can never produce an error since it doesn’t know what authorizer is being produced. Really the API needs reworking…

Reply:
Reported at: https://github.com/Azure/go-autorest/issues/580